src/SerdeError.php

```diff
@@ -7,5 +7,6 @@
 enum SerdeError
 {
     case Missing;
+    case Uninitialized;
```

Comment:
Maybe this could be Missing, but at least from reading the code again this is more readable & explicit.

Reply:
Honestly... I don't think it's correct to put Uninitialized into SerdeError at all.  It's not an error; it's a surrogate sentinel value for the data, which is not an error case.  I think it probably needs its own enum.

Reply:
Ok, created `SerdeSpecialValue::Uninitialized` WDYT?